### PR TITLE
Add missing repository root files

### LINKED ISSUE #126: Include missing top-level repo files
This repo is missing the following:

- AUTHORS
- CODE_OF_CONDUCT.md
- CONTRIBUTORS.md
- CONTRIBUTING.md

I suggest we don't include changelog for this repository, since it contains generated code only

### PR DESCRIPTION
**Pull request by Andy-Grigg**
Closes #126 

I have omitted changelog, since it doesn't make sense to maintain one for this repo.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This repository is not under active development. It is only for storing the code generated from the API
```
[the remark this answers is not in the document]
Comment (Andy-Grigg) [reply CONTRIBUTING.md:1]: Do you think we should stick with the standard text?

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This is the list of GRANTA MI RecordLists API significant contributors.
```
[the remark this answers is not in the document]
Comment (Andy-Grigg) [reply AUTHORS:1]: Totally missed this. Should this instead be "Granta MI BoM Analytics OpenAPI"...?

[the remark this answers is not in the document]
Comment (da1910) [reply AUTHORS:1]: Eh, possibly? Though maybe it ought to be `Bom Analytics API Client` or something?

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ludovicsteinbach) [reply CONTRIBUTING.md]: Maybe? This reads to me like the repo archived or something to that extent.

**Review by da1910 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Andy-Grigg) [reply CONTRIBUTING.md]: That's a good point. I've updated to include the standard text.

**Review by ludovicsteinbach — APPROVED** [review]

**Pull request merged**